### PR TITLE
solution and test for #21

### PR DESCRIPTION
**Pull request by bettychow**

### CODE REVIEW

File: solutions/21.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  return ( 
+    newArray  
+   );
+  };
+
+module.exports = {
+  largestDesNums
+};
+
+
```
Comment (msach22) [inline solutions/21.js]: remove extra lines

File: solutions/21.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+   let newArray = [];
+  array.sort(function(a,b) { return  b - a });
+
+  for (let i = 0; i<3; i++) {
+    newArray.push(array[i]);
+  }
+
+  return ( 
+    newArray  
+   );
```
Comment (msach22) [inline solutions/21.js]: return newArray;

File: solutions/21.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+const largestDesNums = (array) => {
+   let newArray = [];
+  array.sort(function(a,b) { return  b - a });
+
```
Comment (msach22) [inline solutions/21.js]: fix spacing to be consistent

Comment (msach22) [inline solutions/21.js]: @bettychow please review comments and resolve

**Comment by songz** [comment]
this has been open for too long without fix. Closing this.

**Pull request closed without merging**